lib/io/hyped_adc/src/lib.rs

```diff
@@ -2,7 +2,11 @@
 
 /// ADC trait used to abstract the ADC peripheral
 pub trait HypedAdc {
+    /// Read value from the ADC channel
     fn read_value(&mut self) -> u16;
+
+    /// Get resolution of ADC sensor
```

Comment:
sorry this is really picky
```suggestion
    /// Get resolution of ADC
```